gather.go

```diff
@@ -543,8 +543,8 @@ func (a *Agent) gatherCandidatesRelay(ctx context.Context, urls []*stun.URI) { /
 			var (
 				locConn       net.PacketConn
 				err           error
-				relAddr       string
-				relPort       int
+				relatedAddr   string
```

Comment:
Apologies for the refactor -- the whole "relay" and "raddr" and "rel" drove me nuts. To clarify:
* a _relay_ address is the "allocation" a TURN server opens up for two peers to talk over
* a _related_ address is the local address/port that a machine is using to talk with the _relay_ address

The _relay_ address is critical. That's actually the only information a peer needs. The _related_ address is just additional information that (AFAICT) can be convenient for debugging. But otherwise is non-functional.